src/App.jsx

```diff
@@ -14,57 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [massege, setMassege] = useState('Jam is selected');
```

Comment:
There is a typo in the variable name `massege`. It should be `message` to correctly reflect its purpose and maintain consistency.